lib/oat/serializer.rb

```diff
@@ -65,5 +77,16 @@ def map_property(name)
       property name, value
     end
 
+    def set_serialized(type, id)
+      @context[:_serialized_entities][type] ||= {}
+      @context[:_serialized_entities][type][id] = true
+    end
+
+    def serialized?(type, id)
+      h = @context[:_serialized_entities]
+      h = h[type] if h
```

Comment:
`@context[:serialized_entities] ` is always assigned on `initialize`, so is this `if h` needed?